src/abstractdataframe/io.jl

```diff
@@ -164,128 +167,140 @@ function html_escape(cell::AbstractString)
     return cell
 end
 
-function _show(io::IO, ::MIME"text/html", df::AbstractDataFrame;
-               summary::Bool=true, eltypes::Bool=true, rowid::Union{Int, Nothing}=nothing)
+function _show(io::IO,
+               ::MIME"text/html",
+               df::AbstractDataFrame;
+               summary::Bool=true,
+               eltypes::Bool=true,
+               rowid::Union{Int, Nothing}=nothing,
+               title::String="",
```

Comment:
@ronisbr - the same issue with `String` we have here. It should be `AbstractString`. I assume PrettyTables.jl accepts any string type?

Reply:
Hi @bkamins !

Actually no. PrettyTables only supports String. By the time I changed, the performance (TPFT) due to compilation caching was huge. Thus, I think we should  not change title here.

I might revisit it someday since newer Julia versions should have improvements in this area.

Reply:
That's surprising. How is that possible? Of course if you use a custom string type there will be some recompilation overhead, but I wouldn't expect it to affect performance when passing a `String`.

Reply:
I don't mean to butt in, and I'm very grateful for the attention for details here, but since the original purpose of this PR is to allow html output in DataFrames, maybe it's best to punt other issues (such as CI, compilation, etc) to other/later PRs so that the functionality of this PR gets out there sooner rather than later. 

I am, of course, speaking almost completely out of self interest here 😄

Reply:
Yeah I don't mean it should be blocking, I'm just curious about it. (Anyway what matters is not when we merge the PR but when we tag a new release including it!)

Reply:
Hi @nalimilan !

What I saw (but this was a long time ago, Julia 1.3 maybe?) is that when I used `AbstractString` in a keyword argument in the function that prints the table, Julia was not precompiling it with `String` as an argument. Hence, when I called it the first time, it took a lot of time to print the table. When I changed, it became very fast since the lowered code was cached. I am almost sure it was improved in latest version. I will revisit it sometime.